FrontEnd/FrontEnd.Web/Components/Pages/Users/SingleUser.razor

```diff
@@ -21,7 +21,7 @@ else if (_user == null)
 else
 {
     <FluentStack Orientation="Orientation.Vertical">
-        <FluentPersona Image="@_imageUrl" ImageSize="200px" Style="margin: 0 auto;"/>
+        <FluentPersona Image="@_user.ImageUrl" ImageSize="200px" Style="margin: 0 auto;"/>
```

Comment:
Nice touch. I think this is better than trying to force an image upload feature at this point of time in the project.